src/widgets/ResultRow.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2016
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableOpacity, StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { APP_FONT_FAMILY, SUSSOL_ORANGE, BACKGROUND_COLOR, GREY } from '../globalStyles';
+
+const ResultRow = props => {
+  const { data, renderLeftText, renderRightText, onPress, isSelected, showCheckIcon } = props;
+  const selected = isSelected(data);
+
+  return (
+    <TouchableOpacity
+      style={[localStyles.resultRow, selected && localStyles.selected]}
+      onPress={() => onPress(data)}
```

Comment:
as above, annonymous functions and styles/objects/arrays should be explicitly defined and ideally memoized (`useCallback`) where possible

Reply:
Resolved with e9ac13e